providers/yttcb/clusterbootstrap.yaml

```diff
@@ -204,6 +210,26 @@ metadata:
 #@ end
 
 ---
+#@ if azurefilecsi_configs_exist():
+apiVersion: csi.tanzu.vmware.com/v1alpha1
+kind: AzureFileCSIConfig
+metadata:
+  name: #@ data.values.CLUSTER_NAME
+  namespace: #@ data.values.NAMESPACE
+spec:
+  azureFileCSIDriver:
+    namespace: kube-system
```

Comment:
I might have made a mistake that, we'd better to replace this line with
```
namespace: #@ data.values.NAMESPACE
```
probably we could change them in all three drivers later, sounds if this is an issue, it should be P1~P2